debezium-core/src/main/java/io/debezium/relational/TableId.java

```diff
@@ -145,6 +148,21 @@ public String identifier() {
         return id;
     }
 
+    @Override
+    public List<String> parts() {
+        return Collect.arrayListOf(catalogName, schemaName, tableName);
```

Comment:
If we add the `CONNECTOR` part to the mix, it may eliminate the need for the introduction of the `topic.prefix` property and the `LOGIC_NAME_PLACEHOLDER`. It would be just another variable that could be optionally put anywhere in the `topic.parts` property.

Reply:
Definitely interesting idea for the default naming strategy. It will also remove the need for most regexrouter usescases -like use only table name. But this means that every connector will have its own unique default configuartion of `topic.parts` as some use catalog, some schema, right?